cmd/estuary-shuttle/rpc.go

```diff
@@ -319,61 +321,81 @@ func (s *Shuttle) handleRpcAggregateStagedContent(ctx context.Context, cmd *drpc
 	default:
 		return err
 	case nil:
-		// exists already
+		// if it was aggregated successfully but estuary was not notified,, resend message
+		if p.Active {
+			return s.resendPinComplete(ctx, p)
+		}
 		return nil
 	case gorm.ErrRecordNotFound:
 		// normal case
 	}
 
-	totalSize := int64(len(cmd.ObjData))
 	for _, c := range cmd.Contents {
```

Comment:
nit: can we consider some different var names here than `c` and `cmd.Contents` since they're the `AggregateContent` type? would help readability imo. maybe `for _, aggrContent := range cmd.AggrContents`. or we could rename `AggregateContent` to `ContentMetadata`, but plurality gets confusing with that one, so defer to you

Reply:
the naming is ok, the issue I see here is `cmd`, it should have been `aggregate`, so it reads `aggregate.Contents`